app/javascript/src/components/Dashboard/Notes/Pane/Create.jsx

```diff
@@ -6,20 +6,19 @@ import Form from "./Form";
 
 import { NOTES_FORM_INITIAL_FORM_VALUES } from "../constants";
 
-const NewNotePane = ({ fetchNotes, showPane, setShowPane }) => {
+const NewNotePane = ({ showPane, setShowPane }) => {
```

Comment:
The component name and the component file name should be the same. Go through the Slack thread.

Reply:
I have renamed the component name from `NewNotePane` to `Create` to match the file name. 

Here I avoided renaming the `Create.jsx` file to `NewNotePane` to avoid unnecessary namespacing (as the file is already within `Notes\Pane`). 

I also imported this `Create` component as `CreatePane` within the `Notes\index.jsx` like so:

> `import CreatePane from "./Pane/Create";` 

and added the component as 

> `<CreatePane setShowPane={setShowCreatePane} showPane={showCreatePane} />` 

by updating the state variable and setter function associated with the component.

But I am doubtful whether this naming `Create` and `CreatePane` is creating any ambiguity. Do I needed to use the word `Add` over `Create` here.

Please add your suggestions.

Reply:
@sruthy-j-mallya `Add` keyword should be preferred when we are showing some message etc to the end user. End user needn't be technical enough to understand difference between `create` and `add`. But in our codebase, `create` makes more sense here.

Also `Create` and `CreatePane` doesn't create any ambiguity. You are importing `CreatePane` from `"./Pane/Create"`, where the path already contains the `Pane` keyword.